johnwayodi/sm-api
# #161455586 Rework Models,Tests and Views

#### What does this PR do?
Rework on Models, Views, and Tests based on feedback
#### Description of Task to be completed?
1. Add email property to the user model.
2. Add more validation checks to all the `post` methods.
3. Add check on minimum stock and quantity in the request before making a sale record.
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api.git`
2. Ensure you are in the cloned repo, **_sm-api_** by default.
3. Create a virtual environment using virtualenv and `pip install -r requirements.txt`
3. `git checkout ch-restructure-project-161455586`
4. Run `pytest` to test application.
#### Any background context you want to provide?
Changes are in response to the feedback.
#### What are the relevant pivotal tracker stories?
#161455586
